### PR TITLE
Semantics for CupertinoTabBar

### LINKED ISSUE #19921: CupertinoTabBar accessibility
Needs selected state, the tab hint and x of y hint

### PR DESCRIPTION
**Pull request by xster**
Fixes #19921

### CODE REVIEW

File: packages/flutter/lib/src/cupertino/bottom_tab_bar.dart [hunk 1]
```diff
@@ -156,25 +156,30 @@ class CupertinoTabBar extends StatelessWidget implements PreferredSizeWidget {
...
-                padding: const EdgeInsets.only(bottom: 4.0),
-                child: new Column(
-                  mainAxisAlignment: MainAxisAlignment.end,
-                  children: <Widget> [
-                    new Expanded(child: new Center(child: items[index].icon)),
-                    items[index].title,
-                  ],
+            child: new Semantics(
+              selected: active,
+              hint: 'tab, ${index + 1} of ${items.length}',
```
Comment (jonahwilliams) [inline packages/flutter/lib/src/cupertino/bottom_tab_bar.dart]: I think this needs to be localized somehow, I don't have a good solution here.

Comment (xster) [reply packages/flutter/lib/src/cupertino/bottom_tab_bar.dart]: We'd have to do #13452 first

Comment (goderbauer) [inline packages/flutter/lib/src/cupertino/bottom_tab_bar.dart]: What about localization? We have a localized string for this in the MaterialLocalizations, not sure if that one can be reused here?

Comment (xster) [reply packages/flutter/lib/src/cupertino/bottom_tab_bar.dart]: We can't import package material. We'd have to solve #13452 first

Comment (goderbauer) [reply packages/flutter/lib/src/cupertino/bottom_tab_bar.dart]: Understood. Can you add a TODO here to remind us that this needs localization?

Comment (xster) [reply packages/flutter/lib/src/cupertino/bottom_tab_bar.dart]: sg, done

**Review by jonahwilliams — APPROVED** [review]
LGTM

**Review by goderbauer — APPROVED** [review]
LGTM

**Pull request merged**